.github/workflows/ci.yml

```diff
@@ -24,8 +25,8 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version-file: 'pyproject.toml'
-          cache: 'poetry'
+          python-version-file: "pyproject.toml"
+          cache: "poetry"
 
       - name: Install python dependencies
         run: poetry install
```

Comment:
Looks like `black --check .` passes on this branch 🎉 Please can we take the opportunity to uncomment this in CI? (Sorry, github won’t let me comment on the right bit)

Reply:
If we uncomment it can there be a config for a line length of at least 92.

Reply:
We changed live-api from 79 to 88. python-template is currently using 88. Why would we switch to 92?

Reply:
I didn't realise we'd settled on 88, bit of a shame but better than 79

Reply:
Have enabled black (and isort!)